main.tf

```diff
@@ -93,6 +93,7 @@ resource "lacework_external_id" "aws_iam_external_id" {
 }
 
 resource "lacework_integration_aws_agentless_scanning" "lacework_cloud_account" {
+  // TODO: add depends on
```

Comment:
PS. Remove these if they are not needed.

Reply:
good catch thank you, removed